patterns/hidden-404.php

```diff
@@ -5,10 +5,17 @@
  * Inserter: no
  */
 ?>
-<!-- wp:heading {"className":"has-text-align-center"} -->
-<h2 class="has-text-align-center"><?php echo esc_html_x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ); ?></h2>
+
+<!-- wp:heading {"style":{"typography":{"fontSize":"clamp(4rem, 40vw, 20rem)","fontWeight":"400","lineHeight":"1.25"}},"className":"has-text-align-center"} -->
+<h2 class="has-text-align-center" style="font-size:clamp(4rem, 40vw, 20rem);font-weight:400;line-height:1.25"><?php echo esc_html( _x( '404', 'Error code for a webpage that is not found.', 'twentytwentythree' ) ); ?></h2>
 <!-- /wp:heading -->
-<!-- wp:paragraph {"align":"center"} -->
-<p class="has-text-align-center"><?php esc_html_e( 'This page could not be found. Maybe try a search?', 'twentytwentythree' ); ?></p>
+
+<!-- wp:paragraph {"align":"center","fontSize":"small"} -->
+<p class="has-text-align-center"><?php esc_html_e( 'This page could not be found.', 'twentytwentythree' ); ?></p>
 <!-- /wp:paragraph -->
-<!-- wp:search {"label":"Search","showLabel":false,"width":50,"widthUnit":"%","buttonText":"Search","buttonUseIcon":true,"align":"center"} /-->
+
+<!-- wp:spacer {"height":"var(--wp--preset--spacing--100)"} -->
+<div style="height:var(--wp--preset--spacing--100)" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
```

Comment:
The spacer block is showing an invalid content error for me:

![image](https://user-images.githubusercontent.com/1645628/185184461-72df0cb5-61a2-4447-bb34-469a6a88079c.png)


If I replace the markup with the below, it seems to fix the error:

```
<!-- wp:spacer {"height":"var(--wp--preset--spacing--100)"} -->
<div style="height:var(--wp--preset--spacing--100)" aria-hidden="true" class="wp-block-spacer"></div>
<!-- /wp:spacer -->
```

Reply:
Ok, I have added the `aria-hidden="true"` now invalid content error notice is not showing but the gap is not showing for me in the block editor.

Reply:
Hmm, I see this too. I have seen this recently elsewhere, although I thought it had been fixed or it was a problem with my environment. I think we should open an upstream issue for this in [Gutenberg](https://github.com/WordPress/gutenberg/issues). Would you like to handle this? Or I can do it, if you prefer.

Otherwise, I think this PR is good to bring in 🎉

Reply:
Ok, you can open an upstream issue for this in Gutenberg.

Reply:
Do you have Gutenberg enabled?
If you create this block using WordPress 6.0.1 only, the aria-hidden is not included and the spacing presets do not work.

Reply:
I can see the same as @carolinan now, it seems I wasn't on Gutenberg trunk when I commented earlier (I thought I'd seen it working 😄). Sorry for the confusion! This is working in the editor:

<img width="765" alt="image" src="https://user-images.githubusercontent.com/1645628/185627609-eb2b70ba-32b1-48eb-a2b8-6caf856b9191.png">

Sounds like you probably need to activate Gutenberg if you also weren't seeing the aria-hidden attribute (and I'd recommend pulling the latest version from trunk).

Reply:
Sorry for the confusion @mikachan unfortunetly I was similar as yours. 😄

By the way!

> I know this is a unique size that is unlikely to be used anywhere else, so perhaps we should leave the settings in the pattern in this case. Also happy for this to happen in a separate PR!

I also agree with you.